oauth2client/django_util/__init__.py

```diff
@@ -0,0 +1,70 @@
+"""Utilities for the Django web framework
+
+Provides Django views and helpers the make using the OAuth2 web server
+flow easier. It includes a ``required`` decorator to automatically ensure
+that user credentials are available.
+
+
+Configuration
+=============
+
+First, add the helper to your INSTALLED_APPS:
+
+INSTALLED_APPS = (
+    # other apps
+    "oauth2client.django_util"
+)
+
+To configure, you'll need a set of OAuth2 client ID from the
+`Google Developer's Console <https://console.developers.google.com/project/_/\
+apiui/credential>`__.
+
+.. code-block:: python
+   :caption: settings.py
+   :name: settings-file
+
+   GOOGLE_OAUTH2_CLIENT_SECRET=client-secret.json file
+
+Or,
+
+.. code-block:: python
+   :caption: settings.py
+
+
+   GOOGLE_OAUTH2_CLIENT_ID=client-id-field
+   GOOGLE_OAUTH2_CLIENT_SECRET=client-secret-field
+
+By default, the default scopes for the required decorator only contains the (email) scopes. You can
+change that default in the settings.
+
+GOOGLE_OAUTH2_SCOPES = ('https://www.googleapis.com/auth/calendar',)
+
+
+then in your urls.py
+
+.. code-block:: python
+   :caption: urls.py
+
+   import oauth2client
+   urlpatterns += [url(r'^oauth2/', include(oauth2client.django_util.site.urls))]
```

Comment:
I wish installed_apps did a lot of things it doesn't do. But yeah if you just make a new Django project you'll note it includes the urls from django.contrib.admin in this way, other plugins do it too.